Dockerfiles/midstream/unit_tests.py

```diff
@@ -80,10 +80,24 @@ def test_negative_image_to_test_not_set(self):
                       "Result code not found in %s" % message)
         self.assertEqual(102, result.returncode)
         self.assertTrue(os.path.exists(OUTPUT_DIR+".errormessage"))
-    
+
+    # Run with a test-operator which fails the deprecated image check in the
+    # bundle image validation job
+    def test_default_negative(self):
+        self.env["IMAGE_TO_TEST"] = "quay.io/cvpops/test-operator:v1.0-16"
```

Comment:
This bundle seems to have an invalid ocp version support label `v4.5-test` which results in an empty list returned from Pyxis. I've made an updated version of this bundle image with the correct `v4.5` label and pushed it to `quay.io/cvpops/test-operator:test-default-negative-v1`